jvector-twenty/src/main/java/io/github/jbellis/jvector/vector/SimdOps.java

```diff
@@ -26,24 +26,50 @@
 import jdk.incubator.vector.ShortVector;
 import jdk.incubator.vector.VectorMask;
 import jdk.incubator.vector.VectorOperators;
+import jdk.incubator.vector.VectorSpecies;
 
+import java.nio.ByteOrder;
 import java.util.List;
 
-final class SimdOps {
+class SimdOps {
```

Comment:
Minor suggestion: Maybe having SimdOps as a singleton class would not be a bad idea